src/chains/migrations/0038_auto_20221024_1543.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.0.5 on 2022-10-24 15:43
+
+from django.apps.registry import Apps
+from django.contrib.auth.models import Group, Permission
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+def create_support_group(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    support_group, created = Group.objects.get_or_create(name="support")
+    permission = Permission.objects.filter(codename="change_chain")
```

Comment:
Where is this permission defined? It should be on `Meta` for the `Chain` model: https://docs.djangoproject.com/en/4.1/topics/auth/customizing/#custom-permissions